client/my-sites/plans-v2/plans-filter-bar/index.tsx

```diff
@@ -28,16 +28,20 @@ import type { Duration, ProductType } from '../types';
 import './style.scss';
 
 interface Props {
-	duration: Duration;
-	productType: ProductType;
-	onDurationChange: Function;
-	onProductTypeChange: Function;
+	showDurations?: boolean;
```

Comment:
Again, this feels like it could be rigid if we add more controls, but since we only have two for now, I opted not to over-plan.